docs/marketplace/dev/feature-integration-guides/202106.0/merchant-portal-core-feature-integration.md

```diff
@@ -146,15 +208,45 @@ Open access to the Merchant Portal login page by default:
 $config[AclConstants::ACL_DEFAULT_RULES] = [
   [
     'bundle' => 'security-merchant-portal-gui',
-    'controller' => 'login',
-    'action' => 'index',
+    'controller' => '*',
+    'action' => '*',
     'type' => 'allow',
   ],
 
 ];
 ```
+### 3) Add translations
+Add Merchant Portal translations:
+
+1. Append glossary according to your configuration:
+
+**data/import/common/common/glossary.csv**
+```
+mail.merchant.restore_password.subject,Neues Passwort für Ihren Spryker Merchant Portal Account,de_DE
+mail.merchant.restore_password.subject,New password for your Spryker Merchant Portal account,en_US
+mail.merchant.restore_password.text,"""Es wurde ein neues Passwort für Ihren Spryker Merchant Portal Account angefordert. Wenn Sie nicht um um eine Neueinrichtung des Passworts gebeten haben, ignorieren Sie bitte diese Email. Klicken Sie auf den folgedenen Link, um Ihr Passwort zu ändern:""",de_DE
+mail.merchant.restore_password.text,"""Somebody has requested a new password for your Spryker Merchant Portal account. If it's not you, just ignore this email. If it's you, please follow the link bellow:""",en_US
+mail.merchant.restore_password.label,Passwort ändern,de_DE
+mail.merchant.restore_password.label,Change Password,en_US
+mail.trans.merchant.restore_password.subtitle,"Somebody has requested a new password for your Spryker Merchant Portal account. If it was not you, please ignore this email. If it was you, please follow the instructions in  the button bellow",en_US
+mail.trans.merchant.restore_password.subtitle,"Es wurde ein neues Passwort für dein Merchant Portal-Konto angefordert. Wenn du es nicht warst, kannst du diese E-Mail ignorieren. Wenn du es warst, klicke bitte den Knopf unten an:",de_DE
+mail.trans.merchant.restore_password.title,New password for your Spryker Merchant Portal account,en_US
+mail.trans.merchant.restore_password.title,Neues Passwort für dein Spryker Merchant Portal-Konto,de_DE
+```
+
+2. Import data:
+
+```bash
+console data:import glossary
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the configured data has been added to the `spy_glossary` table in the database.
+
+{% endinfo_block %}
 
-### 3) Set up transfer objects
+### 4) Set up transfer objects
```

Comment:
Sections go in the wrong order, please see https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides for the correct one, and re-arrange.